--- a/src/sdl_handmaiden.c	2014-12-27 18:41:22.601020440 +0100
+++ b/src/sdl_handmaiden.c	2014-12-27 18:41:42.501019664 +0100
@@ -152,8 +152,8 @@
 	buf->pitch = 0;
 }
 
-internal void sdl_resize_texture_buf(SDL_Window * window,
-				     SDL_Renderer * renderer,
+internal void sdl_resize_texture_buf(SDL_Window *window,
+				     SDL_Renderer *renderer,
 				     struct sdl_texture_buffer *texture_buf)
 {
 	int height, width;
@@ -185,8 +185,8 @@
 	}
 }
 
-internal void sdl_blit_bytes(SDL_Renderer * renderer, SDL_Texture * texture,
-			     const SDL_Rect * rect, const void *pixels,
+internal void sdl_blit_bytes(SDL_Renderer *renderer, SDL_Texture *texture,
+			     const SDL_Rect *rect, const void *pixels,
 			     int pitch)
 {
 	if (SDL_UpdateTexture(texture, rect, pixels, pitch)) {
@@ -197,7 +197,7 @@
 	SDL_RenderPresent(renderer);
 }
 
-internal void sdl_blit_texture(SDL_Renderer * renderer,
+internal void sdl_blit_texture(SDL_Renderer *renderer,
 			       struct sdl_texture_buffer *texture_buf)
 {
 	SDL_Texture *texture = texture_buf->texture;
