--- a/src/sdl_handmaiden.c	2014-12-27 14:08:01.901659978 +0100
+++ b/src/sdl_handmaiden.c	2014-12-27 14:08:08.317659728 +0100
@@ -56,8 +56,8 @@
 	SDL_Event *event;
 };
 
-internal void sdl_resize_texture_buf(SDL_Window * window,
-				     SDL_Renderer * renderer,
+internal void sdl_resize_texture_buf(SDL_Window *window,
+				     SDL_Renderer *renderer,
 				     struct sdl_texture_buffer *texture_buf)
 {
 	int height, width;
@@ -86,8 +86,8 @@
 	resize_pixel_buffer(pixel_buf, height, width);
 }
 
-internal void sdl_blit_bytes(SDL_Renderer * renderer, SDL_Texture * texture,
-			     const SDL_Rect * rect, const void *pixels,
+internal void sdl_blit_bytes(SDL_Renderer *renderer, SDL_Texture *texture,
+			     const SDL_Rect *rect, const void *pixels,
 			     int pitch)
 {
 	if (SDL_UpdateTexture(texture, rect, pixels, pitch)) {
@@ -98,7 +98,7 @@
 	SDL_RenderPresent(renderer);
 }
 
-internal void sdl_blit_texture(SDL_Renderer * renderer,
+internal void sdl_blit_texture(SDL_Renderer *renderer,
 			       struct sdl_texture_buffer *texture_buf)
 {
 	SDL_Texture *texture = texture_buf->texture;
